spec/models/proxies/government_proxy_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe Proxies::GovernmentProxy do
+  before :each do
+    @proxy = Proxies::GovernmentProxy.new
+    @status = {}
+    @headers = {}
+    Proxies::GovernmentProxy::PROXY_BASE_PATH = '/proxy/gov/'.freeze
+  end
+
+  describe '#rewrite_response' do
```

Comment:
Can you add some line breaks between the blocks to make it consistent with other specs? (Sorry about the style comment, this should be picked up by Rubocop).